packages/angular/src/generators/library/lib/update-project.ts

```diff
@@ -1,188 +1,13 @@
-import {
-  addProjectConfiguration,
-  generateFiles,
-  joinPathFragments,
-  offsetFromRoot,
-  readProjectConfiguration,
-  removeProjectConfiguration,
-  Tree,
-  updateJson,
-} from '@nrwl/devkit';
-import { getRelativePathToRootTsConfig } from '@nrwl/js';
-import { replaceAppNameWithPath } from '../../../utils/cli-config-utils';
+import { offsetFromRoot, Tree, updateJson } from '@nrwl/devkit';
 import { NormalizedSchema } from './normalized-schema';
 import { updateNgPackage } from './update-ng-package';
-import { join } from 'path';
 
 export async function updateProject(
   host: Tree,
   options: NormalizedSchema['libraryOptions']
 ) {
-  createFiles(host, options);
   updateProjectTsConfig(host, options);
```

Comment:
Do we still need to update the tsconfig here? The template should have all that's needed, isn't it?